### DISCUSSION TITLE
Email bounced and cannot be forwarded to your mailbox

### DISCUSSION THREAD

**Discussion by Heziode** [simple-login/app#1279] [Q&A] [answered]
I got this message when an email is send from `laposte.net` to my custom alias: "Email bounced and cannot be forwarded to your mailbox"

![CleanShot 2022-09-13 at 14 25 41@2x](https://user-images.githubusercontent.com/16735076/189900498-98b3aeaa-52b9-4d5c-a551-edda2ab994d7.png)

I use the cloud solution (not self-hosted), with a Proton Visionary plan.

How can I fix this issue? (that blocks me a bit 😅)

**Comment by Heziode** [comment]
![CleanShot 2022-09-13 at 15 03 31@2x](https://user-images.githubusercontent.com/16735076/189908436-24c5dc69-943f-46c5-8202-248e73e72965.png)

It is weird. When we use a custom domain on the cloud version (`app.simplelogin.io`), the generated reply is from `@simplelogin.co` and not our custom domain.

@nguyenkims It is a bug or it is the expected behavior ?

**Reply by nguyenkims** [reply]
Yes emails are sent from "reverse alias" and reverse alias always use simplelogin.co as this domain has the best deliverability.

**Reply by Heziode** [reply]
Ok, so this is the expected behavior, and the fix is to add `simplelogin.co` into whitelist and excluded from filters checks of the mail provider (Infomaniak in my case).

**Comment by Heziode** [comment, marked as answer]
Well... that's crap. I spent a long time with Infomaniak technical support.
The mega-problem I encountered is that even if you configure a whitelist and filters to ignore the check of emails with Infomaniak, they use upstream filters for spam risks. As a result, some emails (such as emails for tracking packages for example) are bounced away by the system.

To sum up, it is therefore impossible to use Infomaniak with SimpleLogin, otherwise there is a risk of having emails that are not delivered.

This annoys me a lot, as I wanted to do without ProtonMail because I find it too annoying to use in the long term. 

---

So, @nguyenkims, use of `simplelogin.co` for reverse alias is not the best for deliverability  😅

**Reply by nguyenkims** [reply]
We have encountered issues with Infomaniak in the past as well and despite several goings back and forth with Infomaniak team, the issue doesn't seem to be fixed ... If possible, I'd recommend to use another mailbox service in the meantime. We'll try to get in touch again with Infomaniak again but I'm not sure if the bug will be definitely fixed this time.
